web/src/api/ipfix.ts

```diff
@@ -53,6 +54,20 @@ export enum FlowDirection {
   Inner = '2'
 }
 
+export const getFlowDirectionDisplayString = (value: FlowDirection, t: TFunction) => {
+  return value === FlowDirection.Ingress
+    ? t('Ingress')
+    : value === FlowDirection.Egress
+    ? t('Egress')
+    : value === FlowDirection.Inner
```

Comment:
Indeed, but since the data come from JSON, anything can happen here.
For example, I sometime got number instead of string in `FlowDirection` field.
See https://github.com/netobserv/network-observability-console-plugin/pull/435#issuecomment-1903899792

Reply:
see https://github.com/netobserv/flowlogs-pipeline/blob/40b69485049482aee71b55a5b7b0fa52cd49576e/pkg/pipeline/transform/transform_network_direction.go#L62